requirements/specifications/lifecycle/presentation.md

```diff
@@ -0,0 +1,249 @@
+| `fullscreen` | The app is displayed such that the dimensions fill the entire screen                                                    |
+| `offscreen`  | The app is has it's graphics surface attached, but not displayed on the screen at the moment, e.g. scrolled out of view |
+| `scaled`     | The app is displayed at a size smaller than the entire screen but at least 25% of the width or height                   |
+| `thumbnail`  | The app is displayed at a size smaller than 25% of the width or height of the entire screen                             |
+| `loading`    | The platform is displaying a loading screen while the app prepares to be activated                                      | 
+| `none`       | The app does not have it's graphics surface attached to the screen                                                      |
+
+**TODO**: an app could be offscreen & scaled. If a scaled or thumbnailed app 
+goes offscreen, it's now offscreen. 
+
+### 4.1. Display vs Lifecycle
+
+Each Lifecycle state only supports certain display states: 
+
+| Lifecycle      | Supported Displays                               |
+|----------------|--------------------------------------------------|
+| `initializing` | `none`, `loading`                                |
+| `active`       | `fullscreen`, `scaled`, `thumbnail`, `offscreen` |
+| `running`      | `none`, `loading`                                |
+| `suspended`    | `none`, `loading`                                |
+| `sleeping`     | `none`, `loading`                                |
+
+See [Picture-in-picture](#9-picture-in-picture-video) and [Background 
+Audio](#8-background-audio) for exceptions to this. 
+
+## 5. Loading Screen
+In order to manage user expectations, Firebolt platforms **MAY** display 
+loading screens to end users when an app is going to be activated. Loading 
+Screens may be rendered either by the platform, the app, or both, depending on 
+Capability configuration. 
+
+Proposal: 
+
+If an app has the `xrn:firebolt:capability:presentation:loading-screen` 
+capability, it **MAY** be made visible any time from the beginning of 
+activate() transition and **MUST** be made visible no later than the end. 
+
+If an app does not have the 
+`xrn:firebolt:capability:presentation:loading-screen` capability, it **MUST** 
+be made visible at the end of the activate() transition. 
+
+### 5.1. Platform-provided Loading Screen
+
+Most apps will leverage a platform-provided loading screen. 
+
+If an app provides the `xrn:firebolt:capability:presentation:loading-screen` 
+capability, then the platform **MAY** use the app-provided loading screen, in 
+which case, the rest of the section does not apply. 
+
+The loading screen **SHOULD** include a loading image referenced in the app's 
+manifest and cached on the device. 
+
+The loading screen **MUST** be displayed when the user attempts to launch the 
+app. 
+
+The loading screen **MUST** stay displayed until the app becomes active, or 
```

Comment:
We should support hybrid. Even if they use an app provided loading screen, it will still take time to get the app provided loading screen downloaded and rendered. So maybe this should read the loading screen is displayed until the app becomes active or the app loading screen is ready, or launching is cancelled.